IanGRap/Squish
# Bugfix: Double jump when moving into a stack of blocks

Contributors: Austin

Goal: Fix Issue #16 (https://github.com/IanGRap/Squish/issues/16)

Important Files Changed:
- PlayerGrounded.cs
    - Made the radius of the overlap sphere serialized so that it can be changed within the prefab
- Player Prefab: Lowered the Overlap Sphere Radius so that the player no longer thinks it is grounded when hugging certain parts of the blocks.

How to test:
- Load the development branch in sprint 2 demo scene and follow the reproduce case laid out in Issue 16 until you see the bug
- Load this branch and try to reproduce the bug. You should ideally be unable to
